reports/blueprint/report.tex

```diff
@@ -296,16 +305,79 @@ \section{Budget} % Ethan
 additional funding. The fully expanded component BOM for the PCB is provided in
 Appendix \ref{appendix:bom}.
 
-% include some preamble wrt board quantities and BOM table
-% consider ActiveBOM from Altium for this.
-% BOM probably takes too many pages, will need to attenuate
-\subsection{Materials and Supplies}
 \subsection{Contributions From Other Sources}
 There are no outside contributions at the time of writing this report. However,
-the team will reach out should additional funding be required.
+the team will reach out should additional funding be required. 
 
 \section{Potential Problems and Mitigation Strategies}
 % Ethan (Make a single table)
+There are two main categories for which technical problems can arise in this
+project. Issues are likely to come in the PCB design and in the FPGA Real Time
+Logic (RTL) code. Software remains less of a concern as compile times are
+instantaneous when compared to FPGA synthesis or lead time of a new hardware
+revision. Software can still be a source of potential problems, however,
+hardware or FPGA failure are of greater severity. Given below is a table
+summarizing possible issues in the PCB and FPGA along with ways to reduce their
+possibility or recover from them.
+
+\begin{table}[h!]
+    \caption{Potential Problems and Mitigation}
+    \begin{tabularx}{\textwidth}{p{2in}|p{2in}|p{2in}}
+        \textbf{Problem Description} & \textbf{Possibility Reduction} & \textbf{Course of Recovery} \\
+        \hline
+        USB FIFO chip on the PCB does not work or the FPGA cannot successfully
+        interface with it.
+
+        & Design review from all group members and close attention to reference
+        designs. PCB layout suited to the high speed control signals of the
```

Comment:
> PCB layout suited to the high speed control signals of the chips

Not sure what you mean by this

Reply:
It means that the copper traces on the board are connected in a way that can support the high speed signals we are sending. Ie if the traces were of different widths or differed in length, it would effect prop speed and signal integrity.

Reply:
We can rephrase I just did not want to add too much to the table. it is already killing a page

Reply:
Yeah you can say Design PCB to support high control signal speeds

Reply:
I reworked this a bit but believe we should still use the term PCB layout (as this is the nomenclature that most accurately describes how the copper is poured on the PCB). I think it will make more sense to a reader aware of PCBs then just "design"